docs/devices/9290030674.md

```diff
@@ -106,7 +106,7 @@ It's not possible to read (`/get`) or write (`/set`) this value.
 Value can be found in the published state on the `motion_sensitivity` property.
 To read (`/get`) the value publish a message to topic `zigbee2mqtt/FRIENDLY_NAME/get` with payload `{"motion_sensitivity": ""}`.
 To write (`/set`) a value publish a message to topic `zigbee2mqtt/FRIENDLY_NAME/set` with payload `{"motion_sensitivity": NEW_VALUE}`.
-The possible values are: `low`, `medium`, `high`.
+The possible values are: `very low`, `low`, `medium`, `high`, `very high`.
```

Comment:
Same as https://github.com/Koenkk/zigbee2mqtt.io/pull/1626/files#r990817047

Reply:
Forgot this comment, just noticed https://github.com/Koenkk/zigbee-herdsman-converters/pull/4759. This part is automatically updated if exposes is updated so this pr is not needed.